clean.py

```diff
@@ -13,11 +13,11 @@
             dnames.remove('node_modules')
 
 
+subprocess.check_call('python -m pip uninstall -y jupyterlab'.split(), cwd=here)
```

Comment:
I think we should allow this to pass if jupyterlab isn't currently installed.

Reply:
When I tested it locally, pip returned zero for a package that was not installed.

Reply:
https://github.com/pypa/pip/blob/6278c6c37181ae061bc5f28280e86b690f32fa06/tests/functional/test_uninstall.py#L510-L529